metatron-app/metatron-discovery
# F#429 preparation changes file upload api

### Description
preparation file upload is using asynchronous API

**Related Issue** : 
[#429](https://github.com/metatron-app/metatron-discovery/issues/429)

### How Has This Been Tested?
1. create a dataset
2. file upload
3. check the uploaded grid
4. if the dataset is created, that is ok

### Types of changes



### Checklist:



### Additional Context
